faq.md

```diff
@@ -1,45 +1,634 @@
+
+### Which .NET versions are supported?
+
+- **OpenTK 4.x+** runs on .NET Core 2.x, .NET Core 3.x, and .NET 5+.
+
+- **OpenTK 3.x** runs on .NET Framework 4.x.
+
+### Which version of OpenTK should I use?
+
+If you’re able to use **.NET Core or .NET 5+**, you should use .NET Core or .NET 5+, and then use the latest version of **OpenTK 4** or higher with it.
+
+If you’re stuck on the older .NET Framework 4.x, you will need to use OpenTK 3.x with it.  OpenTK 4.x+ is *not* compatible with .NET Framework, and there are no plans to support newer releases of OpenTK on .NET Framework.
+
+### What’s the status of OpenTK 5.0?
+
+OpenTK 5.0 is currently in development, and is not yet ready for release.  We have a [TODO list](https://github.com/opentk/opentk/issues/1207) tracks its current status.  We’d welcome any help, if you’d like to contribute!
+
+### Which release is the official release?
+
+- https://github.com/opentk/opentk/ is the official source-code repository for OpenTK.  There are also several related projects managed by [the OpenTK team](https://github.com/opentk/), including the [GLControl](https://github.com/opentk/GLControl) and [GLWpfControl](https://github.com/opentk/GLWpfControl) and [ObjectTK](https://github.com/opentk/ObjectTK).
+- https://www.nuget.org/packages/OpenTK/ is the official Nuget package for OpenTK.  Similarly, there are [other Nuget packages](https://www.nuget.org/profiles/OpenToolkit) for the related projects.
+
+All other distributions of OpenTK are not officially supported by the OpenTK team.  In particular:
+
+- The Xamarin/Microsoft OpenTK package is a fork of a _very_ old version of OpenTK 1.x.  **The OpenTK team does not support the Xamarin code.**
+- The OpenTK.NETCore package (now unlisted) is a fork of OpenTK 3.  .NET Core projects should use OpenTK 4+ instead.
+
+### Which versions of OpenGL are supported?
+
+OpenTK primarily supports **OpenGL 3+** and **OpenGL ES 2+** — in other words, *modern* OpenGL.  The fixed-function pipeline (FFP) — a.k.a. “classic OpenGL” — can be made to work, but it is not directly supported.
+
+### What about UI frameworks?
+
+OpenTK can be used without a UI framework:  It is capable of spawning a window and taking input directly, which is often sufficient for many programs like games.
+
+But if you need to integrate OpenTK with an existing UI, it includes some packages to do so:
+
+- **OpenTK 4.x** has —
+  - A WinForms-compatible [GLControl](https://github.com/opentk/GLControl) Nuget package.
+  - A WPF-compatible [GLWpfControl](https://github.com/opentk/GLWpfControl) Nuget package.
+
+- **OpenTK 3.x** contains its own WinForms-compatible `GLControl` component.
+
+### How fast is OpenTK?
+
+For a .NET library, OpenTK is *very* fast.  OpenTK 3 and 4 use hand-optimized IL assembly to minimize overhead when calling OpenGL functions, and OpenTK 5 is planned to use the new C# function pointers to reduce overhead even further.
```

Comment:
`to reduce overhead even further.` is misleading as it will have the same overhead as the optimized IL (and maybe even slightly worse as we are not optimizing at IL anymore, but we will try to avoid that).

Reply:
I replaced it with `use the new C# function pointers to accomplish the same thing.`  I think that reads better.